examples/federated-learning/tff/nvflare/client2-deployment.yml

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
This is basically the same Deployment as client1. If we need these two to be different, please use Kustomize so we start from a common base.

Reply:
Done